extensions-contrib/kafka-emitter/src/main/proto/DruidSegmentEvent.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.druid.emitter.kafka.proto";
+option java_outer_classname = "DruidSegmentEventMessage";
+
+/* Druid segment Event used by Druid to publish first level segment information.
+ * The message will be consumed by segment processing app. */
+message DruidSegmentEvent {
+    // Datasource, e.g. telemetry_metrics_storage
```

Comment:
we can remove this comment, I don't think this needs explaining in the Druid codebase.